peps/pep-8107.rst

```diff
@@ -0,0 +1,208 @@
+All active Python core team members are eligible to vote. Active status
+is determined as :pep:`described in PEP 13 <13#membership>`
+and implemented via the software at `python/voters <https://github.com/python/voters>`_
+[#note-voters]_.
+
+Ballots will be distributed based on the `Python Voter Roll
+<https://github.com/python/voters/blob/master/voter-files/>`_ [#note-voters]_
+for this election.
+
+While this file is not public as it contains private email addresses, the
+`Complete Voter Roll`_ by name will be made available when the roll is
+created.
+
+Election Implementation
+=======================
+
+The election will be conducted using the `BetterVoting
+<https://bettervoting.com>`__ service.
+
+.. attention::
+   This election will be the first where voting is conducted, and
+   results are calculated using
+   `Multi-winner Bloc STAR <https://www.starvoting.org/multi_winner>`__
+   as `approved by the core developers <https://discuss.python.org/t/64971>`__
+   and `codified <https://github.com/python/peps/pull/3958/files>`__
+   into :pep:`13`.
+
+
+Configuration
+-------------
+
+Create a `new election <https://bettervoting.com/new_election>`__.
+
+Poll or Election?: ``Election``
+
+Title?: ``2026 Python Steering Council Election``
+
+Restricted?: ``Yes``
+
+Contact Email: ``psf-election@python.org``
+
+Choose Voters: ``Email List``
+
+This will initialize the election and you will be forwarded to the election admin page.
+Further configuration is required.
+
+Click the pencil icon next to the election name on the admin.
+
+Election Description: ``Election for the Python steering council, as specified in PEP 13. This is steering council election for the 2026 term.``
+
+Enable Start/End Times?: ``Check this box``
+
+Time Zone: ``Midway Island, Samoa``
+
+Start Date: ``11/28/2025, 01:00 AM``
+
+End Date: ``12/13/2025, 01:00 AM``
+
+Click "Save".
+
+Click "Extra Settings"
+
+Check "Randomize Candidate Order".
+
+Ensure "Show Preliminary Results" is unchecked.
+
+Check "Confirm That Voter Read Instructions".
+
+Ensure "Make Election Publicly Searchable" is unchecked.
+
+Ensure "Set Number of Rankings Allowed" is unchecked.
+
+Click "Save".
+
+* Voting is not open to the public, only those on the `Voter Roll`_ may
+  participate. Ballots will be emailed when voting starts.
+* Candidates are presented in random order, to help avoid bias.
+
+Races
+-----
+
+Add Race
+
+Race Title: ``2026 Python Steering Council``
+
+Race Description: ``Rate candidates for the Python Steering Council``
+
+How many Winners?: ``Basic Multi-Winner``
+
+Number of winners: ``5``
+
+Which Voting Method: ``STAR Voting``
+
+Candidates (add each candidate):
+
+* TBD
+* TBD
+* TBD
+* TBD
+* TBD
+
+Now, use "Cast test ballot" section to preview the ballot and resolve any misconfigurations.
+
+Voters
+------
+
+Enter voter data using Email list from `Voter Roll`_ repository.
+
+Results
+=======
+
+Of NN eligible voters, MM cast ballots.
+
+The top five vote-getters are:
```

Comment:
I would just say "The five winners are:". STAR Voting uses a automatic runoff in each round to ensure that each winner selected has a majority support from voters who showed a preference. This will usually be the same as the top-five score getters but not always.

Reply:
I agree (I've also never been a fan of the 'vote-getter' phrasing).
```suggestion
The five winners are:
```